package com.bms.Connection;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class ProvideConnection {
	 public static Connection connectionKodi()throws ClassNotFoundException, SQLException {

	        Class.forName("com.mysql.cj.jdbc.Driver");

	        Connection con = DriverManager.getConnection("jdbc:mysql://localhost:3306/pentagon_evening","root","tiger");

	        return con;
	    }

}
